pkg/security/securitycontextconstraints/provider_test.go

```diff
@@ -608,27 +608,6 @@ func TestValidatePodSecurityContextSuccess(t *testing.T) {
 }
 
 func TestValidateContainerSecurityContextSuccess(t *testing.T) {
-	var notPriv bool = false
-	defaultPod := func() *api.Pod {
```

Comment:
Here is the identical function that will be used instead: https://github.com/openshift/origin/blob/07ec3d29aac2c2af47246be40ac7af3ee248fa32/pkg/security/securitycontextconstraints/provider_test.go#L892-L911